R/load_fotmob.R

```diff
@@ -19,16 +19,10 @@
     fotmob_urls$id
   )
 
-  fp <- purrr::possibly(
-    .file_reader,
-    quiet = FALSE,
-    otherwise = data.frame()
-  )
-
-  res <- purrr::map_dfr(urls, fp)
+  res <- purrr::map_dfr(urls, .file_reader)
```

Comment:
there's a handful of drive by changes in this script. nothing major

Reply:
here, i realized that `.file_reader` is already a `tryCatch()` wrapper, so we don't need `possibly()`. We may consider not suppressing warnings for when there is an error in `.file_reader`, given how [we've had some recent discussion ](https://github.com/JaseZiv/worldfootballR/issues/244)about showing errors/warnings to the user

Reply:
Yeah nice one